### ISSUE TITLE
add seeking for tracks backed by uade

### ISSUE THREAD

**Issue by ciros88** [aargirakis/BZRPlayer#267]

**Comment by aargirakis** [comment]
That's a hard one, most of the tracks from UADE/EagelPlayer doesn't know the length of the track.

**Comment by ciros88** [comment]
I have been open this issue because the same tracks that now are played with uade without been seekable, before was played by libxmp and they were seekable

**Comment by aargirakis** [comment]
Which ones were seekable?

**Comment by ciros88** [comment]
[Uploading pru2.zip…]()

**Comment by aargirakis** [comment]
Yeah, but that's pretty much a packed mod, a pretty simple format.

**Comment by ciros88** [comment]
i will provide you a list of tracks seekable by libxmp but not by uade

**Comment by ciros88** [comment]
what i've done is:
as starting point, i've added the 'libxmp' dir contained in the pack u sent me and added it to 2.0.33.Alpha, then played all the tracks and removed the seekable ones. then saved the playlist (with all the track are backed by uade) with name 'unseekable by uade on current.m3u'.
then opened it on 2.0.20.Alpha, removed the skipped and the unseekable one so only the tracks seekable by libxmp remains: i've saved this playlist then with a texteditor i've edited all the #EXTINF: time values to 0. finally, i have opened this last playlist on 2.0.33.Alpha: all the tracks songlength is unknown. i've named this playlist as 'seekable by libxmp on old release but not by uade on current release.m3u'.
summarizing:
all the tracks in 'seekable by libxmp on old release but not by uade on current release.m3u' playlist are seekable by libxmp in 2.0.20.Alpha but not by uade on 2.0.33.Alpha. you could use it as starting reference for trying to make uade tracks seekable

[playlists.zip](https://github.com/aargirakis/BZRPlayer/files/10234581/playlists.zip)

**Comment by ciros88** [comment]
seeking has been implemented in the upcoming 2.0.82 with switch to matti's uade fork.
for those tracks not having a lenght, the relative seeking it is still available (thru ← → arrows)
